src/mono/netcore/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.cs

```diff
@@ -751,6 +751,7 @@ internal class RuntimeConstructorInfo : ConstructorInfo
 #pragma warning disable 649
         internal IntPtr mhandle;
         private string? name;
+        private string? toString;
         private Type? reftype;
```

Comment:
The runtime depends on this layout, so if new fields are added, they should be added at the end.

Reply:
That explains all the crashes :) I will add a comment to the class

Reply:
We usually add something like:
```
		#region Sync with ReflectionAssembly in object-internals.h
...
                 #endregion
```